alicloud/resource_alicloud_instance_test.go

```diff
@@ -1354,6 +1354,7 @@ func TestAccAliCloudECSInstancePrepaidAll(t *testing.T) {
 		},
 	})
 }
+
 func TestAccAliCloudECSInstanceDataDisks(t *testing.T) {
```

Comment:
=== RUN   TestAccAliCloudECSInstanceBasic
    provider_test.go:143: Skipping unsupported region eu-central-1. Supported regions: [cn-shenzhen cn-shanghai cn-beijing cn-qingdao cn-hangzhou cn-hongkong us-west-1 ap-southeast-1]. Using ap-southeast-1 as this test region
    provider_test.go:88: Skipping testing classic resources, otherwise setting environment parameter 'ENABLE_CLASSIC_NETWORK'.
--- SKIP: TestAccAliCloudECSInstanceBasic (0.00s)

Test ignored.
PASS

=== RUN   TestAccAliCloudECSInstanceVpc
--- PASS: TestAccAliCloudECSInstanceVpc (716.34s)
PASS

=== RUN   TestAccAliCloudECSInstancePrepaid
--- PASS: TestAccAliCloudECSInstancePrepaid (731.82s)
PASS

=== RUN   TestAccAliCloudECSInstancePrepaidAll
--- PASS: TestAccAliCloudECSInstancePrepaidAll (199.26s)
PASS

=== RUN   TestAccAliCloudECSInstanceDataDisks
--- PASS: TestAccAliCloudECSInstanceDataDisks (117.28s)
PASS

=== RUN   TestAccAliCloudECSInstanceTypeUpdate
--- PASS: TestAccAliCloudECSInstanceTypeUpdate (169.23s)
PASS

=== RUN   TestAccAliCloudECSInstanceSpotInstanceLimit
--- PASS: TestAccAliCloudECSInstanceSpotInstanceLimit (112.34s)
PASS

=== RUN   TestAccAliCloudECSInstanceMulti
--- PASS: TestAccAliCloudECSInstanceMulti (112.38s)
PASS

    provider_test.go:154: Skipping unsupported region eu-central-1. Supported regions: [cn-shanghai cn-beijing cn-zhangjiakou].
--- SKIP: TestAccAliCloudECSInstanceHpcCluster (0.00s)

Test ignored.
PASS

=== RUN   TestAccAliCloudECSInstanceSecondaryIps
--- PASS: TestAccAliCloudECSInstanceSecondaryIps (107.07s)
PASS

=== RUN   TestAccAliCloudECSInstanceSecondaryIpCount
--- PASS: TestAccAliCloudECSInstanceSecondaryIpCount (165.99s)
PASS

=== RUN   TestAccAliCloudECSInstance_DeploymentSetID
--- PASS: TestAccAliCloudECSInstance_DeploymentSetID (138.59s)
PASS

=== RUN   TestAccAliCloudECSInstance_StatusUpdated
--- PASS: TestAccAliCloudECSInstance_StatusUpdated (270.60s)
PASS

=== RUN   TestAccAliCloudECSInstanceMetadataOptions
--- PASS: TestAccAliCloudECSInstanceMetadataOptions (161.12s)
PASS

=== RUN   TestAccAliCloudECSInstanceIpv6AddressesCount
--- PASS: TestAccAliCloudECSInstanceIpv6AddressesCount (267.38s)
PASS

=== RUN   TestAccAliCloudECSInstanceIpv6Addresses
--- PASS: TestAccAliCloudECSInstanceIpv6Addresses (183.20s)
PASS

=== RUN   TestAccAliCloudECSInstance_OperatorType
--- PASS: TestAccAliCloudECSInstance_OperatorType (313.91s)
PASS

=== RUN   TestAccAliCloudECSInstance_AutoSnapshotPolicyId
--- PASS: TestAccAliCloudECSInstance_AutoSnapshotPolicyId (135.46s)
PASS

=== RUN   TestAccAliCloudECSInstanceSystemDisk
    provider_test.go:149: Skipping unsupported region eu-central-1. Supported regions: [cn-hangzhou]. Using cn-hangzhou as this test region
--- PASS: TestAccAliCloudECSInstanceSystemDisk (269.18s)
PASS

=== RUN   TestAccAliCloudECSInstanceMaintenance
    provider_test.go:149: Skipping unsupported region eu-central-1. Supported regions: [cn-hangzhou]. Using cn-hangzhou as this test region
--- PASS: TestAccAliCloudECSInstanceMaintenance (162.77s)
PASS

=== RUN   TestAccAliCloudECSInstanceDedicatedHostId
--- PASS: TestAccAliCloudECSInstanceDedicatedHostId (121.44s)
PASS

=== RUN   TestAccAliCloudECSInstance_LaunchTemplate
--- PASS: TestAccAliCloudECSInstance_LaunchTemplate (214.24s)
PASS